code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -593,6 +608,25 @@
 			udder.trans_type_to(G, milk_type, rand(5, 10))
 			return
 
+	if(has_toxingland)
+		var/obj/item/reagent_containers/glass/Test = attacking_item
```

Comment:
Meaningful var name, "Test" isn't one of those

Reply:
ctcontainer stands for CarpoToxin Container